CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -140,7 +136,7 @@ public function buildQuickForm() {
     $listingTypeField = $this->addRadio('listing_type', E::ts('Type of Service Listing'), $serviceListingOptions, $attr);
     $organizationNameField = $this->add('text', 'organization_name', E::ts('Organization Name'), $attr);
     $this->add('email', 'organization_email', E::ts('Organization Email'));
-    $this->add('text', 'website', E::ts('Website'), NULL, TRUE);
+    $this->add('text', 'website', E::ts('Website'), NULL);
```

Comment:
Doesn't this make it non required? won't that then mean we need to make adjustments to the tpl to have the required * on it?

Reply:
Yes, they don't want this field to be required anymore. Just removing the TRUE does the needful and removes out the *